# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: 9f55c4ae34ed
# Full Hash: 9f55c4ae34edac0c72687198415b2c075889f9ea
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 1
# Description:
#   Bug 1654112 - Remove a bogus assertion. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113831
# ==============================================================================

diff -r 2392ed5dd474 -r 9f55c4ae34ed dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Thu Apr 29 18:25:50 2021 +0000
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Thu Apr 29 18:25:47 2021 +0000
@@ -58,7 +58,6 @@
  */
 WebrtcAudioConduit::~WebrtcAudioConduit() {
   CSFLogDebug(LOGTAG, "%s ", __FUNCTION__);
-  MOZ_ASSERT(NS_IsMainThread());
   MOZ_ASSERT(!mSendStream && !mRecvStream,
              "Call DeleteStreams prior to ~WebrtcAudioConduit.");
 }
